libraries/dictionary/test/dictionaryTypes.spec.ts

```diff
@@ -471,4 +472,34 @@ describe('Dictionary Types', () => {
 			});
 		});
 	});
+	describe('Meta', () => {
+		it('Can accept non-nested values', () => {
+			const meta = { a: 'string', b: 123, c: true };
+			expect(DictionaryMeta.safeParse(meta).success).true;
+		});
+		it('Can accept nested values', () => {
+			const singleNested = { a: 'string', b: 123, c: true, nested: { d: 'asdf' } };
+			const doubleNested = { a: 'string', b: 123, c: true, nested: { d: 'asdf', nested2: { e: 'asdf' } } };
+			expect(DictionaryMeta.safeParse(singleNested).success).true;
+			expect(DictionaryMeta.safeParse(doubleNested).success).true;
+		});
+		it('Can accept arrays of strings', () => {
+			const meta = { a: 'string', b: 123, c: true, array: ['asdf', 'qwerty'] };
+			expect(DictionaryMeta.safeParse(meta).success).true;
+		});
+		it('Can accept arrays of numbers', () => {
+			const meta = { a: 'string', b: 123, c: true, array: [123, 456, 789] };
+			expect(DictionaryMeta.safeParse(meta).success).true;
+		});
+		it('Cannot accept arrays of booleans', () => {
+			// This constraint feels a bit arbitrary but I can't imagine a clear use case for this.
```

Comment:
As commented before, it's unclear "why", so arbitrary or not, behooves explanation

Reply:
So my thinking here is: 

There are only two allowed values (true and false), so an array of booleans only communicates information based on the length of the array, and the position of each Boolean value in the array. If you build an array where the position of each boolean value matters, then that introduces a lot of ambiguity about the meaning of each position in the array, and it would be prone to typos where the values are in the wrong position. I don't want to enable that kind of ambiguity. If what you want is to represent a sequence where only the order matters, that is still accomplishable by a string or number array and those are not limited to Boolean states.

In the end though, if a good use case is presented we can change this decision.

Reply:
that adds clarity, for sure. thanks.
is this rationale documented anywhere I'm not currently aware of?